machine_learning/DegradationModel.py

```diff
@@ -1246,7 +1246,7 @@ def cc_voltage(self, params, training = True):
     def cv_voltage(self, params, training = True):
```

Comment:
why do we even have this function? 

A lot of these functions have been deleted. why are they not deleted here?

Reply:
I barely touched this file. This is the old DegrdationModel

Reply:
ok. permission to delete this file. I have saved it in a separate branch